mlem/contrib/kubernetes/resources.yaml.j2

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: mlem-{{ image_name }}-app
+  labels:
+    name: mlem-{{ image_name }}-app
```

Comment:
make it a separate field. it may default to field

Reply:
it might even be a list?

Reply:
and everywhere else you used this one field

Reply:
this was due yes, made a new one called `{{ namespace }}` now